### DISCUSSION TITLE
Hey

### DISCUSSION THREAD

**Discussion by zyadhpl** [community/community#77444] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

hello, i'm new here someone have any idea for new things to do?

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the Community Discussions @glocktbh  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by zyadhpl** [reply]
Hey @queenofcorgis Thanks for that, tell me about helping, there is someone who needs help in javascript? How i can know about that, i've seen before something but no one pop up to me.

**Reply by queenofcorgis** [reply, marked as answer]
You're welcome 😃 If you're looking to help with a particular topic I recommend using the search bar to find people looking for assistance or discussion in that area.

**Comment by sandipanrakshit34** [comment]
hello
